backend/src/test/java/de/sambam/gamenighttracker/service/UserServiceTest.java

```diff
@@ -133,15 +133,15 @@ public void userHasNoPlayerReturnsEmptyListTest() {
     @DisplayName("no gameList in user with id 1 addNewGame should return empty optional")
     public void addNewGameWithNoGameListTest() {
         //GIVEN
-        when(userDb.findById(userId)).thenReturn(Optional.of(
+        when(userDb.findByUserName(username)).thenReturn(Optional.of(
                 User.builder()
                         .id("1")
-                        .userName("admin")
+                        .userName("sambam")
                         .build()));
 
         Game gameToAdd = Game.builder().name("Monopoly").build();
         //WHEN
-        Optional<Game> newGame = userService.addNewGame(gameToAdd, userId);
+        Optional<Game> newGame = userService.addNewGame(gameToAdd, username);
 
         //THEN
         assertTrue(newGame.equals(Optional.empty()));
```

Comment:
Verify that it has been saved to the db?

Reply:
done ^^'